styleguide-app/Examples/SegmentedControl.elm

```diff
@@ -160,18 +161,33 @@ buildRadioOptions content =
             , label = label
             , value = value
             , idString = String.fromInt value
+            , tooltip =
+                [ Tooltip.plaintext text
+                , Tooltip.open (currentlyHovered == Just value)
+                , Tooltip.fitToContent
+                , Tooltip.onHover
```

Comment:
I've noticed that in most places, we're using `Tooltip.smallPadding`. Perhaps it's worth checking if we need that?
we use it on the `sidebar tabs` and the `expand/collapse buttons`